s4e-web/src/app/form/check-box-list/check-box-list.component.scss

```diff
@@ -1 +1 @@
-@import "../../../scss/panel";
+@import "../../../scss/settings";
```

Comment:
I see a lot of such imports. Most of them are the only line of a component's CSS. I suppose you are using it in such way to encapsulate their styles. Have you maybe wonder if it would be easier for you to start working on global CSS without importing them into specific files?

Reply:
Never mind, @michal-szostak said that you talked already about this issue... :P